src/components/menu/utils/OutsideClick.tsx

```diff
@@ -0,0 +1,62 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+
+export interface IOutsideClickHandlerProps {
+    onOutsideClick: (e: MouseEvent) => void;
+    useCapture?: boolean;
+}
+
+export default class OutsideClickHandler extends React.Component<IOutsideClickHandlerProps> {
```

Comment:
Rename file please to `OutsideClickHandler`

Reply:
Renamed